### DISCUSSION TITLE
MooseFS Pro License

### DISCUSSION THREAD

**Discussion by o-krussow** [moosefs/moosefs#577] [General]
I'm not sure this is the right place to ask, but I've emailed the contact email a few times trying to get in contact with someone about getting a pro license but haven't ever heard back. Are you guys still selling licenses?

I appreciate the help.

**Comment by footlooseboss** [comment]
> Original ticket started on 2022-12-31:
> 
> Hello
> 
> Do you offer a semi-pro (restricted/minimal, perhaps just HA &/ erasure coding) MooseFS Pro license for clumsy home users?
> 
> Cheers
> 
> F

> 2023-01-31
> Re: homelab
> Hello,
> 
> Thank you for contacting us! I am happy that you are interested in MooseFS Pro and take it into a consideration as a solution. I am sure that MooseFS Pro will meet your requirements.
> 
> Please excuse us for a delay in reply – unfortunately your inquiries was improperly filtered as spam in our support system and we didn't see them earlier. I am very sorry for this situation and inconvenience it could have caused and I am sure it won't happen again since we have already whitelisted your domain.
> 
> I am sure you already know MooseFS advantages, so won't spam you with tons of text and descriptions.
> 
> Please let me know what is your necessary RAW disk space of your planned cluster? I will be happy to prepare a quote for you based on this information with special discount for home users.
> Please could you also share with me some information on your planned MooseFS Pro use-case? Are you familiar with MooseFS already? What storage solution are you currently using?
> 
> In the mean time, I would like to describe how MooseFS Pro licensing works. Our pricing model is based on the raw disk space of the cluster: the license is perpetual, valid up to a certain major version (e.g. 4.x), so license fee is a one-time payment for licensed storage space. Technical support is renewed yearly (annual payment). With technical support, apart from our help and expertise knowledge of our engineers, you get access to our packages repository and updates.
> 
> We are flexible when it comes to pricing, and MooseFS Pro is a cost-effective solution: the more storage space you require, the greater discount can be provided – since it grows with raw space of the cluster. So as your cluster grows, the unit storage price (i.e. price per 1 TiB of raw storage space) decreases. For license expansions you pay only for extra space, and we continue or increase the discount.
> 
> Also, if you would like to run several clusters, in the quote we sum up their raw spaces in order to give you the best discount.
> 
> If you would be more interested about some MooseFS Pro insights / use-cases and technical aspects, attached please also find a presentation which may help here and a hardware guide.
> 
> Looking forward to hearing from you.
> 
> Thank you,
> 
> Best regards,
[REDACTED]

**Comment by chogata** [comment]
Hello @o-krussow 
As far as I know this situation is now resolved. Your original e-mail dropped into the wrong ticket queue, but it was then found and someone has contacted you. We are very sorry for the inconvenience.

**Reply by o-krussow** [reply]
That is correct, thank you!
